src/components/chat/chat_channels.jsx

```diff
@@ -51,7 +59,7 @@ const ChatChannels = React.createClass({
 
   getInitialState() {
     return {
-      currentChannel: ChatChannelsStore.getCurrentChannel(),
+      currentChannel: this.getParams().roomSlug,
```

Comment:
Does `getParams()` return at least an empty object (`{}`)? I just want to make sure we don't trigger a `cannot get property "roomSlug" of undefined` error if `getParams()` returns undefined.